book/04-git-server/sections/gitlab.asc

```diff
@@ -24,8 +24,7 @@ For more information read the https://gitlab.com/gitlab-org/gitlab-foss/-/blob/m
 ==== Administration
 
 GitLab's administration interface is accessed over the web.
-Simply point your browser to the hostname or IP address where GitLab is installed, and log in as the admin user.
-The default username is `admin@local.host`, and the default password is `5iveL!fe` (which you must change right away).
+Simply point your browser to the hostname or IP address where GitLab is installed, and log in as the `root` user. The password will depend on your installation type but by default, Omnibus GitLab automatically generates a password for and stores it to /etc/gitlab/initial_root_password for at least 24 hours. Follow the documentation for more details.
```

Comment:
Hi! We like to keep this source code to the one-sentence-per-line format, it makes reviewing diffs a lot easier.

```suggestion
Simply point your browser to the hostname or IP address where GitLab is installed, and log in as the `root` user.
The password will depend on your installation type but by default, Omnibus GitLab automatically generates a password for and stores it to /etc/gitlab/initial_root_password for at least 24 hours.
Follow the documentation for more details.
```